documentation/templates/theme/theme_template.md

```diff
@@ -1,7 +1,22 @@
-"As a [user role], I want to [user story] so that [benefit]. By [action], we can [project objective]. This will enable us to [project outcome]. By following [methodology/principles], we can quickly and effectively [project deliverables], setting us up for [long-term benefit]."
+Themes to build a tutorial web page:
```

Comment:
add links to documentation to readme under "Your plan should replace the content below"